src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/notification/CreateClaimRespondentNotificationHandler.java

```diff
@@ -69,9 +69,9 @@ private CallbackResponse notifyRespondentSolicitorForClaimIssue(CallbackParams c
     public Map<String, String> addProperties(CaseData caseData) {
         return Map.of(
             CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
-            RESPONDENT_SOLICITOR_NAME, getPartyNameBasedOnType(caseData.getApplicant1()),
-            RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getApplicant1()),
+            RESPONDENT_SOLICITOR_NAME, "TBC: Respondent solicitor name (need to add in case data?)",
```

Comment:
I'd leave this one as it was, only updating applicant to respondent, because that's how we do it on all notifications. We'll change it everywhere once we get actual solicitors names